README.md

```diff
@@ -96,6 +97,45 @@ If you plan to make contributions to OpenWhisk, we recommend either a Mac or Ubu
 * [Setup Mac for OpenWhisk](tools/macos/README.md)
 * [Setup Ubuntu for OpenWhisk](tools/ubuntu-setup/README.md)
 
+### Lean Setup
+To have a lean setup (no Kafka, Zookeeper and no Invokers as separate entities), consider the following options.
+
+**Docker Compose**
+```
+git clone -b lean https://github.com/kpavel/incubator-openwhisk-devtools.git
+cd incubator-openwhisk-devtools/docker-compose
+make quick-start
+```
+
+For more detailed instructions or if you encounter problems see the [OpenWhisk-dev tools](https://github.com/kpavel/incubator-openwhisk-devtools/tree/lean/docker-compose/README.md) project.
+
+**Native**
```

Comment:
This is a PR to this repo, but pushed down into the native deploy instructions at the appropriate point